.github/workflows/Deploy-prd.yml

```diff
@@ -1,9 +1,12 @@
-name: Deploy
+name: Deploy-prd
```

Comment:
Can we change this to `prod` instead of `prd`? `Prod` is already shortened from `Production`

Reply:
oh some issues-- there's too many places where that affects the code!

Reply:
The NODE_ENV is configured to use that value which is pulled to match the name of the config file

Reply:
Ah ok, sounds good.